src/deploy.sh

```diff
@@ -478,6 +486,23 @@ function modules_install_to()
   cmd_manager "$flag" "$cmd"
 }
 
+# This function will copy and install the compiled dtb files
+# @target Where should the dtb files be installed (1-vm, 2-local or 3-remote)
+function dtb_install()
+{
+  local target="$1"
+  local architecture="${configurations[arch]}"
+
+  case "$target" in
+    1 | 3) # VM_TARGET or REMOTE_TARGET
```

Comment:
That's not how we handle VM... You cannot suppose the VM is running; actually, it is the opposite of that. See how modules are deploys for reference.

Reply:
@rodrigosiqueira this function has a single entry point, which happens after line 105 where the target is checked and the vm is mounted. Am I missing something? my understanding is that the vm will necessarily be running at this point